gradle/libs.versions.toml

```diff
@@ -76,6 +102,15 @@ mockito-inline = { module = "org.mockito:mockito-inline", version.ref = "mockito
 mockito-kotlin = { module = "org.mockito.kotlin:mockito-kotlin", version.ref = "mockitoKotlin" }
 mockito-mockito-core-v3124 = { module = "org.mockito:mockito-core", version.ref = "mockitoMockitoCoreVersion" }
 mockito-mockito-kotlin = { module = "org.mockito.kotlin:mockito-kotlin" }
+mockk = { module = "io.mockk:mockk", version.ref = "mockk" }
+mockk-agent = { module = "io.mockk:mockk-agent", version.ref = "mockkAgent" }
+mockk-android = { module = "io.mockk:mockk-android" }
+mockk-android-v11312 = { module = "io.mockk:mockk-android", version.ref = "mockkAndroidVersion" }
+mockk-android-v1132 = { module = "io.mockk:mockk-android", version.ref = "mockkMockkAndroid" }
+mockk-mockk-android = { module = "io.mockk:mockk-android", version.ref = "mockkAndroid" }
+mockk-v1130 = { module = "io.mockk:mockk", version.ref = "mockkVersion" }
+mockk-v1132 = { module = "io.mockk:mockk", version.ref = "mockkMockk" }
+play-services-auth = { module = "com.google.android.gms:play-services-auth", version.ref = "playServicesAuth" }
```

Comment:
Question: I'm not sure why several versions of mockk are included here?

Reply:
I have no idea, this was all added automatically by my IDE, but after the clean up it should be better now.